### PR TITLE
feat: integrate tx origin blacklist, fix config typos

### PR DESCRIPTION
**Pull request by alexkroeger**
# Description

Integrate a new version of asset-swapper that has a tx origin blacklist option for RFQT.

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

**Comment by alexkroeger** [comment]
deploy staging

File: src/config.ts [hunk 1]
```diff
@@ -189,15 +189,25 @@ export const RFQT_REGISTRY_PASSWORDS: string[] = _.isEmpty(process.env.RFQT_REGI
     ? []
     : assertEnvVarType('RFQT_REGISTRY_PASSWORDS', process.env.RFQT_REGISTRY_PASSWORDS, EnvVarType.JsonStringList);
 
-export const RFQT_API_KEY_WHITELIST: string[] = _.isEmpty(process.env.ALT_RFQ_MM_ENDPOINT)
+export const RFQT_API_KEY_WHITELIST: string[] = _.isEmpty(process.env.RFQT_API_KEY_WHITELIST)
```
Comment (alexkroeger) [inline src/config.ts:192]: found typo, fixed \

File: src/config.ts [hunk 2]
```diff
@@ -189,15 +189,25 @@ export const RFQT_REGISTRY_PASSWORDS: string[] = _.isEmpty(process.env.RFQT_REGI
...
+    ? new Set()
+    : new Set(
+          assertEnvVarType(
+              'RFQT_TX_ORIGIN_BLACKLIST',
+              process.env.RFQT_TX_ORIGIN_BLACKLIST,
+              EnvVarType.JsonStringList,
+          ).map((addr: string) => addr.toLowerCase()),
+      );
+
+export const ALT_RFQ_MM_ENDPOINT: string | undefined = _.isEmpty(process.env.ALT_RFQ_MM_ENDPOINT)
```
Comment (alexkroeger) [inline src/config.ts:210]: found typo, fixed

File: src/config.ts (hunk 1 above)
Comment (PirosB3) [reply src/config.ts:192]: was this broke?

**Review by PirosB3 — APPROVED** [review]

File: src/config.ts (hunk 1 above)
Comment (alexkroeger) [reply src/config.ts:192]: no, they're defined so it doesn't hit this part of the code, but should be fixed anyway

**Pull request merged**